### ISSUE TITLE
pkgr will not install from filesystem if using ~

### ISSUE THREAD

**Issue by dpastoor** [metrumresearchgroup/pkgr#208]
will get an error around "downloading" the package:

```
FATA[0000] missing package                               package=rematchpath="~/clients/metrum/r-snapshots/2019-12-02/bin/macosx/el-capitan/contrib/3.5/rematch_1.0.1.tgz"
```

in reality this is because go does not expand ~ by default, we need to wrap the download string in a call to michellh/homedir to attempt to expand as needed.

**Comment by dpastoor** [comment]
@Dreznel has this been addressed yet, and if so can we please link the PR

**Comment by Dreznel** [comment]
I would assume this is addressed in #243 , which has been merged into #226. My apologies for not linking this issue at the time.

**Comment by Dreznel** [comment]
related: #235
